### PR TITLE
Platir 8744

### PR DESCRIPTION
**Pull request by tumulurik**

### CODE REVIEW

**Review by ichasepucks — APPROVED** [review]
Thanks.  Didn't realize we missed this one.  Are there other spark based recipes that we missed as well?

**Comment by tumulurik** [comment]
Np, don't think we missed any more spark recipes.

**Comment by sathyarun18** [comment]
Thank you for getting this done

**Review by sathyarun18 — APPROVED** [review]

**Comment by jonaphin** [comment]
great work, thank you for taking care of it 🙏😊

File: recipes/feature_pipeline_recipes/pyspark/README.md [hunk 1]
```diff
@@ -24,23 +24,33 @@ script within `/acp-data-services-dsw-reference/bootstrap`.
...
-    i) Supply the pipeline.json as form-data to the engine key
+From the Select Source, in the Runtime drop down select PySpark and make sure the Artifact type drop down has docker 
+selected. 
+Copy paste the appropriate values when running the login.sh script.
+After login is successful, run `build.sh`
+
+```
+sh build.sh
+```
+This pushes the docker image to the ACR for eg: in the case where the Docker host is `v1d2cs4mimnlttw.azurecr.io`
```
Comment (jonaphin) [inline recipes/feature_pipeline_recipes/pyspark/README.md:44]: you may want to make this generic since we're on public github

**Review by jonaphin — COMMENTED** [review]
one small comment, otherwise great, thank you!

**Pull request merged**